src/components/labware/PacbioRunWell.vue

```diff
@@ -122,20 +113,25 @@ const storeWell = computed(() => {
  * @returns {string} - The tooltip for the well.
  */
 const tooltip = computed(() => {
-  return [...storeWell.value.pools, ...storeWell.value.libraries]
-    .map((id) => {
-      return store.tubeContents.find((tubeContent) => id == tubeContent.id).barcode
-    })
-    .filter(Boolean)
-    .join(',')
+  return storeWell.value?.used_aliquots
+    ? [...storeWell.value.used_aliquots]
+        .map(({ source_id, source_type }) => {
+          const type = source_type === 'Pacbio::Pool' ? 'pools' : 'libraries'
```

Comment:
Ill have a look into it it but im not sure about forcefully changing the type because we rely on it in a few places like in the badge of the items the pool/library list and when linking data together.

Reply:
Had a look - we are using that type to find where to look in the store in a few places so it we made this change we would essentially have to do the reverse in some other places to fix it so think its best to leave.

Reply:
Cool, That make sense. No worries 👍